packages/@okta/vuepress-theme-prose/layouts/AuthorizationLayout.vue

```diff
@@ -22,6 +22,20 @@
 export default {
   components: {
     Footer: () => import('../components/Footer.redesign')
+  },
+  mounted: async function() {
+    await (function(apiKey){
+          (function(p,e,n,d,o){var v,w,x,y,z;o=p[d]=p[d]||{};o._q=o._q||[];
+          v=['initialize','identify','updateOptions','pageLoad','track'];for(w=0,x=v.length;w<x;++w)(function(m){
+              o[m]=o[m]||function(){o._q[m===v[0]?'unshift':'push']([m].concat([].slice.call(arguments,0)));};})(v[w]);
+              y=e.createElement(n);y.async=!0;y.src='https://cdn.pendo.io/agent/static/'+apiKey+'/pendo.js';
+              z=e.getElementsByTagName(n)[0];z.parentNode.insertBefore(y,z);})(window,document,'script','pendo');
+              pendo.initialize({
+                  visitor: {},
+                  account: {}
+              });
+          }
+          )('f8bd2822-002a-478f-66a9-0178efd7ee1f');
```

Comment:
Can we put the function in a separate file so we don't have to copy/paste it 3 times?

Reply:
Already done, thanks.